bases/node14.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Node 14",
+
+  "compilerOptions": {
+    "lib": ["es2019", "es2020"],
```

```suggestion
    "lib": ["es2020"],
```

Doesn't es2020 cover everything in es2019?

---

Good catch, I'll fix it tomorrow 😉